operator/config/prometheus/monitor.yaml

```diff
@@ -1,17 +1,16 @@
-
 # Prometheus Monitor Service (Metrics)
 apiVersion: monitoring.coreos.com/v1
 kind: ServiceMonitor
 metadata:
   labels:
     control-plane: controller-manager
   name: controller-manager-metrics-monitor
-  namespace: system
+  namespace: thundernetes-system

```

Comment:
Ah, great catch!.

Reply:
These don't get get added by the Makefile yet. Since the monitoring pieces have dependency on Prometheus CRDs, I'll think of a good way to introduce these into the 'make install-files' actions without a breaking change.

Reply:
also, I think that some breaking changes are fine at this point :D

Reply:
It's more than just a breaking change, since the system wouldn't install without prometheus operator being installed first, so there's a specific make file command that will add them, but it isn't a default action for the `create-install-files`